internal/topology/variables/cluster_variable_defaulting.go

```diff
@@ -49,8 +50,11 @@ func defaultClusterVariables(values []clusterv1.ClusterVariable, definitions []c
 	// - variables with the same name do not have a mix of empty and non-empty DefinitionFrom.
 	valuesIndex, err := newValuesIndex(values)
 	if err != nil {
```

Comment:
TIL: Technically we produce this error during defaulting and validation, so we have two of them. But this gets deduplicated during error aggregation in apierrors.NewInvalid